### ISSUE TITLE
Invalid Api key causes high CPU usage

### ISSUE THREAD

**Issue by shatl** [splitio/.net-client#94]
Following code causes high CPU usage in splitio-net-core 3.3.1

```
var factory = new SplitFactory("REPLACEME", new ConfigurationOptions
{
  FeaturesRefreshRate = 60,
});
var splitClient = factory.Client();
```

**Comment by patricioe** [comment]
Thank you for flagging. We'll take a look.

**Comment by SenhorCastor** [comment]
Hi @shatl . We just released version 3.3.3 with a fix for this issue.

Thanks for flagging!

**Comment by SenhorCastor** [comment]
Going to close for now, feel free to reach out again if needed
